msal-dotnet-articles/advanced/high-availability.md

```diff
@@ -103,19 +103,18 @@ Certificates for the confidential client app must be rotated for security reason
 
 1. Use managed identity
 
-With [managed identity](/entra/msal/dotnet/advanced/managed-identity), trust is established through hosting your app in Azure. There are not secrets to maintain and no certificates to rotate. 
+With [managed identity](./managed-identity.md), trust is established through hosting your app in Azure. There are not secrets to maintain and no certificates to rotate.
 
-2. Use Microsoft.Identity.Web's certificate handling logic
+2. Use the `Microsoft.Identity.Web` certificate handling logic
 
-In web apps and web APIs, use Microsoft.Identity.Web, a higher-level API over MSAL. It handles certificate rotation when the certificate is stored in Azure Key Vault and handles managed identity case as well.
+In web apps and web APIs, use `Microsoft.Identity.Web`, a higher-level API over MSAL. It handles certificate rotation when the certificate is stored in Azure Key Vault and handles managed identity case as well.
 
 Learn more in the [Certificates in Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web/wiki/Certificates#getting-certificates-from-key-vault) guide.
 
 This is the preferred solution for non-Microsoft internal services using ASP.NET Core.
 
-3. (**Internal only**) Rely on [Subject Name/Issuer certificates](./subject-name-and-issuer-authentication.md).
+3. (**Microsoft internal only**) Rely on Subject Name/Issuer certificates.
 
-This mechanism allows Azure AD to identify a certtificate based on SN/I instead of a thumbprint (x5t). It is a stop-gap solution; there are no plans to make it available to non-Microsoft applications. 
+This mechanism allows Azure AD to identify a certtificate based on SN/I instead of a thumbprint (x5t). It is a stop-gap solution; there are no plans to make it available to non-Microsoft applications.
```

Comment:
```suggestion
This mechanism allows Azure AD to identify a certificate based on SN/I instead of a thumbprint (x5t). It is a stop-gap solution; there are no plans to make it available to non-Microsoft applications.
```